Gravity/Gravity.Test.Unit/RsapiDaoUpdateTests.cs

```diff
@@ -372,7 +372,7 @@ public void Update_ChildObject_UpdateInsertNew_IgnoredWithNoRecursion()
 			UpdateObject(objectToUpdate, rdo => true, ObjectFieldsDepthLevel.OnlyParentObject);
 			CollectionAssert.AreEqual(new[] { g2caId, 0 }, objectToUpdate.GravityLevel2Childs.Select(x => x.ArtifactId));
 		}
-
+		/*
```

Comment:
Why was this commented out?

Reply:
@Arithmomaniac This was commented out because when building the solution I get an error "Using the generic type 'WriteResultSet<T>' requires 1 type arguments" on line 407

Reply:
Well, then we need to fix it 😃 Try replacing it with `WriteResultSet<RDO>`.